bond/bond_test.go

```diff
@@ -38,110 +40,95 @@ const (
 			"cniVersion": "%s",
 			"mode": "%s",
 			"failOverMac": 1,
-			"linksInContainer": true,
+			"linksInContainer": %s,
 			"miimon": "100",
-			"mtu": 1400,
+			"mtu": %s,
 			"links": [
 				{"name": "net1"},
 				{"name": "net2"}
 			]
 		}`
 	ActiveBackupMode = "active-backup"
 	BalanceTlbMode   = "balance-tlb"
+	DefaultMTU       = 1400
 )
 
 var Slaves = []string{Slave1, Slave2}
 
 var _ = Describe("bond plugin", func() {
 	var podNS ns.NetNS
+	var initNS ns.NetNS
+	var linkAttrs []netlink.LinkAttrs
```

Comment:
I think you don't need to have linkAttrs at this level. It is only used locally within the beforeEach

Reply:
sure thing :+1: